packages/mobile/src/transactions/TransferConfirmationCard.tsx

```diff
@@ -90,6 +94,13 @@ function InviteSentContent({
         }
       />
       <HorizontalLine />
+      <FeeDrawer
+        currency={CURRENCY_ENUM.DOLLAR}
+        inviteFee={inviteFee}
+        isInvite={true}
+        securityFee={securityFee}
+        totalFee={inviteFee}
```

Comment:
nit: maybe create a variable for total fee? seems like it would be easy to forget that total doesnt just mean invite and then months later we find this and "think woah we accidentally set total just the invite fee!"